lib/diagnostic.js

```diff
@@ -5,17 +5,27 @@
 // the minimum value in an array
 // this callback will be passed by the tests to the reduce method, e.g.
 // `let minimum = array.reduce(min);`
-const min = function (/* previous, current */) {
-
+const min = function (callback, array) {
```

Comment:
This function is being passed into ```reduce()``` as a callback, so its arguments should be the accumulator (we named it ```previous``` in this case), ```current```, and the current working array (omitted here).